BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -1225,10 +1226,11 @@ public void CheckDetonationState(bool separateWarheads = false)
                         else
                         {
                             float optimalDistance = (float)(Math.Max(DetonationDistance, relativeSpeed));
-                            var hitCount = Physics.OverlapSphereNonAlloc(warheadType == WarheadTypes.ContinuousRod? vessel.CoM -= VectorUtils.GetUpDirection(TargetPosition) * (GetBlastRadius() > 0 ? (DetonationDistance / 3) : 5) : vessel.CoM, optimalDistance, proximityHitColliders, layerMask);
+                            Vector3 targetPoint = (warheadType == WarheadTypes.ContinuousRod ? vessel.CoM -= VectorUtils.GetUpDirection(TargetPosition) * (GetBlastRadius() > 0 ? (DetonationDistance / 3) : 5) : vessel.CoM);
```

Comment:
Why is this using `-=` here? Surely this should just be `-`?

Reply:
Hmm. yeah, that should be ``-``.